nautiluszim/scraper.py

```diff
@@ -282,58 +280,16 @@ def check_branding_values(self):
             )
 
     def check_description_length(self):
-        # Checking if the provided input is a file or text
-        if os.path.isfile(os.path.join(os.getcwd(), self.description)):
-            with open(
-                os.path.join(os.getcwd(), self.description), "r", encoding="utf-8"
-            ) as fp:
-                text = ""
-                for line in fp:
-                    line = line.strip()
-                    if len(line) >= 80:
-                        raise ValueError(
-                            f"--The description is greater than 80 characters: {line}"
-                        )
-                    text += line
-                self.description = text
-
-        else:
-            if len(self.description) >= 80:
+        # Checking if the provided input is a greater than 80 characters
```

Comment:
I have removed the support for file to be submitted as description and the function checks whether the description length is less than 80 characters or not

Reply:
scraperlib has the constant for the nb of characters.

and check both in a single method